spec/schemas.spec.js

```diff
@@ -105,6 +105,8 @@ const userSchema = {
     email: { type: 'String' },
     emailVerified: { type: 'Boolean' },
     authData: { type: 'Object' },
+    MFAEnabled: { type: 'Boolean' },
```

Comment:
This should be camelCase to match convention

Reply:
Like this?

` mfaEnabled: { type: 'Boolean' },`

Reply:
yep